bindings/python/bt2/bt2/notification.py

```diff
@@ -434,7 +437,7 @@ def count(self):
     def stream(self):
         stream_ptr = native_bt.notification_discarded_packets_get_stream(self._ptr)
         assert(stream_ptr)
-        return bt2.stream._create_from_ptr(stream_ptr)
+        return bt2.stream._Stream._create_from_ptr(stream_ptr)
```

Comment:
Not sure about this. Why not using the IR domain's `create_stream_from_ptr()`?

Reply:
done.